package.json

```diff
@@ -349,7 +349,12 @@
             ]
           },
           "description": "Specifies one or more URLs that point of template galleries for creating swings."
-        }
+        },
+        "codeswing.themePreview": {
+          "default": false,
+          "type": "boolean",
+          "description": "Use Visual Studio Code theme for the webview pannel (these styles will only be visible when a swing is viewed in VSCode)."
```

Comment:
Can we set the description to the same text you're using in the `README` for the `Codeswing: Theme Preview` setting? (minus the `Defaults to` part). That way we are consistent